From 2de3b96a9d2033e6ee9710d45f582aa5d2b248d9 Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio.traversaro@iit.it>
Date: Sat, 3 Apr 2021 15:26:09 +0200
Subject: [PATCH] Use gettimeofday also on macOS in timer instead of deprecated
 function

---
 ode/src/timer.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ode/src/timer.cpp b/ode/src/timer.cpp
index 4f3434a8..60e0cf80 100644
--- a/ode/src/timer.cpp
+++ b/ode/src/timer.cpp
@@ -176,7 +176,7 @@ double dTimerTicksPerSecond()
 
 #if !defined(PENTIUM) && !defined(WIN32)
 
-#ifndef macintosh
+#if 1
 
 #include <sys/time.h>
 #include <unistd.h>
 
